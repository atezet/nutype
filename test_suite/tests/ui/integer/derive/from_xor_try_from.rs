use nutype::nutype;

#[nutype(derive(From, TryFrom))]
struct Number(i32);

fn main() {}
