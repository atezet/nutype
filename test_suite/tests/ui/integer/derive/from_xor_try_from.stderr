error: There is no need to derive `TryFrom` when `From` is already derived.
       There is a blanket implementation for `TryFrom` in std.
 --> tests/ui/integer/derive/from_xor_try_from.rs:3:23
  |
3 | #[nutype(derive(From, TryFrom))]
  |                       ^^^^^^^
